### ISSUE TITLE
mocha not found on Travis CI

### ISSUE THREAD

**Issue by georgeOsdDev** [mobilusoss/textlint-browser-runner#192]
Travis CI build start failing from #190, #191 

It seems `lerna bootstrap` is not called as `postinstall` after `npm ci`.

Success case:
https://travis-ci.com/github/mobilusoss/textlint-browser-runner/builds/191070080

```
npm ci 
> husky@4.3.0 install /home/travis/build/mobilusoss/textlint-browser-runner/node_modules/husky
> node husky install
husky > Setting up git hooks
CI detected, skipping Git hooks installation.
husky > Done
> husky@4.3.0 postinstall /home/travis/build/mobilusoss/textlint-browser-runner/node_modules/husky
> opencollective-postinstall || exit 0
> root@undefined postinstall /home/travis/build/mobilusoss/textlint-browser-runner
> lerna bootstrap
lerna notice cli v3.22.1
lerna info ci enabled
lerna info Bootstrapping 4 packages
lerna info Installing external dependencies
lerna info Symlinking packages and binaries
lerna success Bootstrapped 4 packages
added 944 packages in 20.434s
```

Fail case:
https://travis-ci.com/github/mobilusoss/textlint-browser-runner/builds/191564613

```
$ npm ci 
added 946 packages, and audited 946 packages in 6s
5 high severity vulnerabilities
To address issues that do not require attention, run:
  npm audit fix
Some issues need review, and may require choosing
a different dependency.
Run `npm audit` for details.
```

**Comment by georgeOsdDev** [comment]
https://github.com/npm/cli/issues/1905

**Comment by georgeOsdDev** [comment]
fixed with #193
